### DISCUSSION TITLE
Hosting in Github pages

### DISCUSSION THREAD

**Discussion by codeWithNoob** [community/community#64396] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

GitHub Pages is known for its reliability and speed. How does GitHub Pages achieve such high performance, and what are some best practices for optimizing a GitHub Pages site for speed?

**Comment by InfiniteCoder100** [comment, marked as answer]
GitHub Pages achieves high performance through various means:

CDN (Content Delivery Network): GitHub Pages uses a global CDN to distribute your site's content, reducing latency for users around the world.

Caching: GitHub Pages caches your site's content on edge servers, ensuring fast access to frequently visited pages.

HTTPS: GitHub Pages provides free HTTPS, which improves security and can also boost SEO rankings.

Automated Builds: GitHub Pages automatically builds your site from source code, optimizing assets like images and CSS during the process.

**Comment by hannelevans** [comment]
GitHub Pages boosts performance with CDN, caching, and automated builds. For better PPC results, check out [Top PPC Agencies In The Uk](https://pearllemon.com/top-ppc-agencies-in-the-uk/).
